lib/elixir/lib/access.ex

```diff
@@ -305,6 +305,32 @@ defmodule Access do
       "could not put/update key #{inspect key} on a nil value"
   end
 
+  @doc """
+  Removes the entry with a given key from a struct, map, or keyword list.
```

Comment:
"thing that implements the Access behavior" - that's what container was supposed to mean but, since we don't  specify anywhere, I agree with you. :) What if we say: "a map, keyword list or any other structure that implements the Access behaviour"?